src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -127,7 +127,7 @@
   <PropertyGroup Condition="'$(IsTestProject)'=='true'">
     <StartWorkingDirectory Condition="'$(StartWorkingDirectory)'==''">$(TestPath)$(DebugTestFrameworkFolder)</StartWorkingDirectory>
     <StartAction Condition="'$(StartAction)'==''">Program</StartAction>
-    <StartProgram Condition="'$(StartProgram)'==''">$(TestProgram)</StartProgram>
+    <StartProgram Condition="'$(StartProgram)'==''">$(TestHostDirectory)$(TestProgram)</StartProgram>
```

Comment:
In the context of CoreFX, you're right.  While they aren't different today, it is still possible for one's host executable to be in a different location than the working directory.